fem/bilininteg.hpp

```diff
@@ -2436,6 +2473,21 @@ class DGTraceIntegrator : public BilinearFormIntegrator
                                    const FiniteElement &el2,
                                    FaceElementTransformations &Trans,
                                    DenseMatrix &elmat);
+
+   using BilinearFormIntegrator::AssemblePA;
+
+   virtual void AssemblePAInteriorFaces(const FiniteElementSpace &fes);
+
+   virtual void AssemblePABoundaryFaces(const FiniteElementSpace &fes);
+
+   virtual void AddMultTransposePA(const Vector &x, Vector &y) const;
+
+   virtual void AddMultPA(const Vector&, Vector&) const;
+
+   static const IntegrationRule &GetRule(Geometry::Type geom, int order,
+                                         FaceElementTransformations &T);
+
+   void SetupPA(const FiniteElementSpace &fes, FaceType type);
```

Comment:
It seems this method is neither defined nor used anywhere.

Reply:
it is defined [here](https://github.com/mfem/mfem/pull/1207/files/45a299b04aa0febbd07a8b6aef928262c4db1035#diff-71b826a0bdf2bdffe818868995499256R134) in ` fem/bilininteg_dgtrace.cpp`, and used in the `AssemblePAInteriorFaces` and `AssemblePABoundaryFaces` methods.

Reply:
Oh OK, I guess I did not look in that file.

If it is used only internally, how about we make it protected?

Reply:
I made it `private`.